### DISCUSSION TITLE
Prisma Unable to establish connect on websocket consumer

### DISCUSSION THREAD

**Discussion by ramcharantw** [prisma/prisma#23279] [Q&A]
### Question

I'm trying to connect to Prisma in Python on getting the message from the WebSocket. unable to establish the connection.

Code
```
def main():
    asyncio.run(obj.connectToWebsocket())

connectToWebsocket(){
    initialization of connect
}

onMessageReceived(){
     database call
}
```

**Logs**
```
EBUG:prisma.engine.utils:Ensuring query engine took: 0.2687s
DEBUG:prisma.engine.query:Running query engine on port 53444
DEBUG:prisma.engine.query:Starting query engine...
DEBUG:prisma.engine.http:Sending GET request to http://localhost:53444/status with content: None
DEBUG:prisma.engine.query:Could not connect to query engine due to RuntimeError; retrying...
DEBUG:prisma.engine.http:Sending GET request to http://localhost:53444/status with content: None
DEBUG:prisma.engine.query:Could not connect to query engine due to RuntimeError; retrying...
{"timestamp":"2024-02-25T20:59:41.602532Z","level":"TRACE","fields":{"message":"Fetched a connection from the pool","duration_ms":53,"item_type":"query","is_query":true,"result":"success"},"target":"quaint::connector::metrics"}
DEBUG:prisma.engine.http:Sending GET request to http://localhost:53444/status with content: None
DEBUG:prisma.engine.query:Could not connect to query engine due to RuntimeError; retrying...
```

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @ramcharantw 

I would recommend opening a GitHub Discussion in Prisma Client Python Repository: https://github.com/RobertCraigie/prisma-client-py/discussions

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
